packages/financial-templates-lib/src/proxy-transaction-handler/DSProxyManager.js

```diff
@@ -146,20 +159,20 @@ class DSProxyManager {
     });
 
     await this.gasEstimator.update();
-    const { receipt } = await runTransaction({
+    const { receipt, returnValue, transactionConfig } = await runTransaction({
+      web3: this.web3,
       transaction: this.dsProxy.methods["execute(bytes,bytes)"](callCode, callData),
-      config: {
-        gasPrice: this.gasEstimator.getCurrentFastPrice(),
-        from: this.account,
-        nonce: await this.web3.eth.getTransactionCount(this.account)
-      }
+      transactionConfig: { gasPrice: this.gasEstimator.getCurrentFastPrice(), from: this.account },
+      availableAccounts: this.availableAccounts
     });
 
     this.logger.info({
       at: "DSProxyManager",
       message: "Executed function on a freshly deployed library, created in the same tx 🤗",
       callData,
-      tx: receipt.transactionHash
+      tx: receipt.transactionHash,
+      transactionConfig,
+      returnValue
```

Comment:
Do we need to `.toString` the `returnValue`?

Reply:
tests I did return a FixedPoint that did not need to be converted. but for consistency, I think we should `toString` it.